### ISSUE TITLE
Build errors on OS X

### ISSUE THREAD

**Issue by szhorvat** [tuttleofx/TuttleOFX#593]
I am trying to build on OS X 10.12 / XCode 9.  I followed the instructions to clone the repo then do `git submodule update -i`.  I do not have Homebrew, only MacPorts. This is why I was trying to build it myself.

I have boost 1.59.0.

After

    ./configure
    make

the build fails at some point.  The only informative errors I can see above are like this one:

```
TuttleOFX/plugins/image/io/AudioVideo/avTranscoder/src/AvTranscoder/properties/PixelProperties.cpp:104:33: error: use of undeclared identifier 'PIX_FMT_PAL'
```

There are several others for `PIX_FMT_RGB`, etc.

Do you know why this happens?

I tried to build with `make -j1`, to see the sequence of errors more clearly, but it still does a parallel build for some reason.

**Comment by cchampet** [comment]
Hello,
Which version of ffmpeg are you using? __PIX_FMT_PAL__ is declared since [labavutil 51.29.0 / 51.19.0](https://github.com/FFmpeg/FFmpeg/blob/596a4cbd0a1bcd2a45fbc97538d4e9bf35bf14a2/doc/APIchanges#L1829)
If this plugin does not matter to you, you can disable it by adding a comment at [this line](https://github.com/tuttleofx/TuttleOFX/blob/develop/plugins/CMakeLists.txt#L4).
Hope this will help ;)

**Comment by szhorvat** [comment]
Thank you for the hint. After disabling this, I hit some other roadblocks during compiling, but before I continue:

All I really want is to use the denoiser plugin with the free version of DaVinci Resolve 14 on OS X. Is this at all possible?  I also tried to use the denoiser from the openfx-misc project, and there I found out that it's quite hard to get things working with Resolve.  If you already know that these plugins don't work with Resolve then I better just stop trying to compile now.
